### ISSUE TITLE
Problem with custom wake word

### ISSUE THREAD

**Issue by caldecy** [Kitt-AI/snowboy#332]
Hi, I have tried changing the file for my personal model downloaded from KittAi and copying it into the /home/pi/alexa-avs-sample-app/samples/wakeWordAgent/ext/resources folder; then I changed the name of the model file in _KittAiSnowyboyWakeWordEngine.cpp_ to the respective name of my personal model. However, after saving the code and running the "make" command, I still have "Alexa" as my Wake Word. 

I've even tried removing the universal Alexa model from the folder and keeping only my personal model in the folder to no avail. 

Can you direct me on where I've gone wrong in my setup? 

_KittAiSnowyboyWakeWordEngine.cpp_ Code: 
```
namespace AlexaWakeWord {

static const std::string  RESOURCE_FILE  = "../ext/resources/common.res";
static const std::string  MODEL_FILE     = "../ext/resources/smartmirror.pmdl";
```

**Comment by chenguoguo** [comment]
Are you using the kitt_ai engine at all?

**Comment by caldecy** [comment]
Yes I am. Just to double check with you if I'm doing it right, here are my steps when running the codes.

```
cd ~/Desktop/alexa-avs-sample-app/samples
cd companionService && npm start

cd ~/Desktop/alexa-avs-sample-app/samples
cd javaclient && mvn exec:exec

cd ~/Desktop/alexa-avs-sample-app/samples
cd wakeWordAgent/src && ./wakeWordAgent -e kitt_ai
```

This is how I should run the codes for Alexa right?

**Comment by chenguoguo** [comment]
This seems OK. I couldn't tell what was the issue without your exact setup. It feels like you were not changing the files that the sample app was actually using (e.g., it probably copied the files to one location and used the files from there). Please also try to re-build all the libraries and binaries.

**Comment by caldecy** [comment]
Yeah I also assumed so because no matter what source file I change it to, it still registers Alexa.
